### PR TITLE
Style + style.view fixes

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/core/infrastructure/model.d.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ import {Event} from './event';
 export declare class Model {
 	constructor();
 
-	static register(name: string, model: Model): Model;
+	static register(name: string, model: {new(): {}}): Model;
```
Comment (etokareva) [inline src/core/infrastructure/model.d.ts:37]: What is the purpose of new()? Don't like this name :-)

Comment (vkorolev) [reply src/core/infrastructure/model.d.ts:37]: It means, that the second arg is a type that has parameterless constructor

File: src/core/style/style.view.js [hunk 2]
```diff
@@ -110,7 +104,12 @@ export class StyleView extends View {
 					const cells = bodyRow.cells();
 					for (let j = 0, cellsLength = cells.length; j < cellsLength; j++) {
 						const cell = cells[j];
-						const column = columns[j];
+						const dataCell = cell.model();
+						if(!dataCell){
```
Comment (vkorolev) [inline src/core/style/style.view.js]: Space after `if` is missing:)

**Review by vkorolev — APPROVED** [review]

File: src/core/infrastructure/model.d.ts (hunk 1 above)
Comment (etokareva) [reply src/core/infrastructure/model.d.ts:37]: @vkorolev thanks

**Pull request merged**